pandas/tests/test_downstream.py

```diff
@@ -46,6 +46,7 @@ def test_dask(df):
         pd.set_option("compute.use_numexpr", olduse)
 
 
+@pytest.mark.xfail(reason="array is read-only")
```

Comment:
Can this test eventually work with cow?

Reply:
yeah we can use literally any other function that doesn't modify inplace, changed to log